tensorflow/contrib/verbs/rdma.cc

```diff
@@ -953,6 +953,16 @@ static void ValidateChecksum(uint64_t expected, uint64_t actual,
   }
 }
 
+// Sync the 'done' operation on the GPU stream, but without all the data
+// copying.
+static void StreamGPUOp(Device* gpu_device,
```

Comment:
If this works for you, OK, but it does seem slightly vulnerable to failure if ever the semantics of CopyGPUTensorToCPU are changed to short circuit the stream event queue for empty tensors.